### PR TITLE
ENG-4954 Add Agenda React MFE

### PR DESCRIPTION
**Pull request by alepintus**

### CODE REVIEW

File: microfrontends/react-widget-agenda/.gitignore [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+# misc
+.DS_Store
+.env.local
+.env.test.local
+.env.production.local
+
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
```
Comment (firegloves) [inline microfrontends/react-widget-agenda/.gitignore:22]: Is this file included in the parent one? Does it contain the same thing?

**Review by firegloves — CHANGES_REQUESTED** [review]

File: microfrontends/react-widget-agenda/.gitignore (hunk 1 above)
Comment (avdev4j) [reply microfrontends/react-widget-agenda/.gitignore:22]: @firegloves I think there is a slight overlap, but I wonder which strategy is the best. Quick though, it's a good idea to split .gitignore files to each component.

File: microfrontends/react-widget-agenda/src/App.js [hunk 2]
```diff
@@ -1,33 +1,80 @@
...
+                });
+                setUsers(updatedUsers);
+                break;
+            default:
+                break;
+        }
+    }
+
+    return (
+        // NOTE: data-theme is for propagate the css variables theme through the shadow dom
```
Comment (avdev4j) [inline microfrontends/react-widget-agenda/src/App.js:55]: @alepintus that might explain why we don't meet the issue here ;)

**Review by avdev4j — APPROVED** [review]

**Review by firegloves — APPROVED** [review]
LGTM

**Pull request merged**